### ISSUE TITLE
Bar error for homogeneous media

### ISSUE THREAD

**Issue by Chenlonw** [ahay/src#113]
I use the following command to draw a figure for a homogeneous media
```
Flow('scale3', 'rho', 'bar mean=y')
Result('rho', ['rho', 'scale3'],'byte mean=y bar=${SOURCES[1]}| grey scalebar=y barlabel="rho(kg/m3)"\
       screenratio=0.3 labelsz=25 labelfat=14 titlesz=8 titlefat=3 axisfat=14 title= color=j')
```
However, the bar value is not the true value in my data set. 
Is there anyone can help me about this?

**Comment by sfomel** [comment]
I am not sure what you are trying to accomplish.

Can you do simply 

`Result('rho', 'grey scalebar=y barlabel="rho(kg/m3)" screenratio=0.3 labelsz=25 labelfat=14 titlesz=8 titlefat=3 axisfat=14 title= color=j')`

**Comment by Chenlonw** [comment]
Hi Fomel,

Thank you for your reply.
I tried to plot a figure for a constant density model. The value of the density is 1500. However, when I tried to plot it with `sfgrey`, the value of the bar gave '-429.5' instead.  I am not sure how to make it right. 

The whole SCons like this
```
from rsf.proj import *

Flow('rho',None,''' spike n1=100 n2=100 mag=1500 ''') 
Result('rho', 'grey scalebar=y barlabel="rho(kg/m3)" screenratio=1 title= color=j')

End()
```
Output:
[out.pdf](https://github.com/ahay/src/files/705715/out.pdf)
